test-new/e2e/GuardianVerifierBase.t.sol

```diff
@@ -612,6 +676,7 @@ abstract contract OwnableValidatorRecovery_AbstractedRecoveryModule_Base is
     ) public {
         IGuardianVerifier.ProofData
             memory proofData = getRecoveryEmailProofData(
```

Comment:
same comment, why do we call this here and on line 704?

Reply:
https://github.com/zkemail/email-recovery/pull/91/files#r2037822289